Source/Core/ManagedArray.js

```diff
@@ -0,0 +1,147 @@
+/*global define*/
+define([
+        './defaultValue',
+        './defined',
+        './defineProperties',
+        './Check'
+    ], function(
+        defaultValue,
+        defined,
```

Comment:
Actually, I'll just do a run and fix all of these myself to save time.  There are a bunch of unused things throughout all of the new files.